### PR TITLE
Tickets/dm 38905

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Review by teweitsai — APPROVED** [review]
Looks good.

File: python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+CMD_ITEMS_TO_IGNORE = frozenset({CommandKey.ID, CommandKey.VALUE})
+
+
+class PneumaticsSimulator:
+    def __init__(self) -> None:
+        self.log = logging.getLogger(type(self).__name__)
+        self.cmd_evt_server = PneumaticsServerSimulator(
+            host=tcpip.LOCALHOST_IPV4,
+            port=5000,
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py:45]: Thanks for noticing this and sorry for not telling you. This is a temporary value. I will discuss this with Patricio tomorrow.

File: python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+            log=self.log,
+            dispatch_callback=self.cmd_evt_dispatch_callback,
+            name="CmdEvtPneumaticsServer",
+        )
+        self.telemetry_server = PneumaticsServerSimulator(
+            host=tcpip.LOCALHOST_IPV4,
+            port=6000,
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py:52]: Thanks for noticing this and sorry for not telling you. This is a temporary value. I will discuss this with Patricio tomorrow.

File: python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+        as well as in the JSON schemas in the schemas directory.
+
+        Parameters
+        ----------
+        data : `dict` of `any`
+            The dict to be verified.
+
+        Returns
+        -------
+        bool:
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/atpneumaticssimulator/pneumatics_simulator.py:113]: Return types don't need to be enclosed in back ticks.

**Pull request merged**